packages/appium/lib/main.js

```diff
@@ -442,7 +442,7 @@ async function main(args) {
     });
   }
 
-  logServerAddress(`${parsedArgs.address}:${parsedArgs.port}${parsedArgs.basePath}`);
+  logServerAddress(`http://${parsedArgs.address}:${parsedArgs.port}${parsedArgs.basePath}`);
```

Comment:
not sure if it is a good idea to hardcode `http`. We could use `parsedArgs.scheme` in order to make it more flexible/error prone

Reply:
yeah good idea. i was looking for some arg like this but couldn't find it. will update.

Reply:
actually @mykola-mokhnach i grepped the codebase for `scheme` but don't see it as an appium arg?

Reply:
ah, looks like that [PR](https://github.com/appium/appium/pull/18562) has never been merged. Sorry for the confusion